specification/regionmove/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview/regionmovecollection.json

```diff
@@ -1873,15 +1935,72 @@
         },
         "additionalInfo": {
           "readOnly": true,
-          "type": "object",
-          "description": "The additional info."
+          "type": "array",
```

Comment:
Convert from object to array is breakingchange

Reply:
As discussed, the service is in private preview, and we are doing this improvements as part of public preview which will be happens by end of next month, probable date 27th September.
Additionally our private preview customer do not use the API's directly instead rely on the portal for the usage.